server/main_test.go

```diff
@@ -168,3 +168,87 @@ func TestStartGameRoute(t *testing.T) {
 	actualResponse := w.Body.String()
 	assert.Equal(t, expectedState, actualResponse)
 }
+
+func TestAddPromptRoute(t *testing.T) {
+	// Test set up
+	router := setupRouter("8080")
+	w := httptest.NewRecorder()
+	data := map[string]string{
+		"groupName":  "addPromptRoute",
+		"playerName": "player1",
+	}
+	jsonData, err := json.Marshal(&data)
+
+	// Create the group
+	req, err := http.NewRequest("POST", "/api/create-game", bytes.NewBuffer(jsonData))
+	assert.Nil(t, err)
+	req.Header.Add("Content-Type", "application/json")
+	router.ServeHTTP(w, req)
+	assert.Equal(t, http.StatusOK, w.Code)
+
+	// Add another player
+	w = httptest.NewRecorder()
+	data = map[string]string{
+		"groupName":  "addPromptRoute",
+		"playerName": "player2",
+	}
+	jsonData, err = json.Marshal(&data)
+	assert.Nil(t, err)
+	req, err = http.NewRequest("POST", "/api/add-player", bytes.NewBuffer(jsonData))
+	req.Header.Add("Content-Type", "application/json")
+	router.ServeHTTP(w, req)
+	assert.Equal(t, http.StatusOK, w.Code)
+
+	// Start the game
+	data = map[string]string{
+		"groupName":  "addPromptRoute",
+		"playerName": "player1",
+	}
+	w = httptest.NewRecorder()
+	jsonData, err = json.Marshal(&data)
+	req, err = http.NewRequest("POST", "/api/start-game", bytes.NewBuffer(jsonData))
+	req.Header.Add("Content-Type", "application/json")
+	router.ServeHTTP(w, req)
+	assert.Equal(t, http.StatusOK, w.Code)
+
+	//Make a post to the add prompts route from player 1, confirm state stays at "Initial Prompt Creation"
+	data = map[string]string{
+		"groupName":  "addPromptRoute",
+		"playerName": "player1",
+		"noun":       "chicken",
+		"adjective1": "snazzy",
+		"adjective2": "portly",
+	}
+	w = httptest.NewRecorder()
+	jsonData, err = json.Marshal(&data)
+	req, err = http.NewRequest("POST", "/api/add-prompts", bytes.NewBuffer(jsonData))
+	req.Header.Add("Content-Type", "application/json")
+	router.ServeHTTP(w, req)
+	assert.Equal(t, http.StatusOK, w.Code)
+
+	actualResponse := w.Body.String()
+	//prompt_uuid := actualResponse  //TODO : will need to get the real uuid from the the response and put it in the expected state
```

Comment:
Should we delete this commented-out line?

Reply:
samesies below

Reply:
Out of curiosity - what would we use the uuid for?

Reply:
I was originally going off the word doc that called for an ID in the prompts struct.  This makes sense if you have players with the same name in the same group, because right now we are just ID'ing based on players name.  I ran into some trouble with the assert equal test on uuid, so then I was extracting the uuid from the response and inserting it into the 'expectedresponse' (sort of negating the point of the test i guess).  A better way would be to make an ID generator and feed the ID to both the prompt and the test or something like that.  Anyway if we have players with the same name in a group we will probably have some other issues so prob not worth worrying about at this time.
TLDR: I will delete and we can worry about the ID stuff for prompts and players and groups later.